### PR TITLE
Adding Lithuania translation

### PR DESCRIPTION
**Pull request by uikolas**
Interesting that nobody have added Lithuania translation of demo application. In Lithuania Symfony is very popular and maybe for new users will be useful to see Lithuania translation.

### CODE REVIEW

File: translations/messages+intl-icu.lt.xlf [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+                <source>help.post_publication</source>
+                <target>Nustatykite datą ateityje, kad suplanuotumėte blogo įrašo paskelbimą.</target>
+            </trans-unit>
+            <trans-unit id="help.post_content">
+                <source>help.post_content</source>
+                <target>Turinį galima formatuoti su Markdown ir HTML žymom.</target>
+            </trans-unit>
+            <trans-unit id="help.comment_content">
+                <source>help.comment_content</source>
+                <target>Komentarai neatitike Elgesio kodekso, bus moderuoti.</target>
```
Comment (javiereguiluz) [inline translations/messages+intl-icu.lt.xlf]: Probably a false positive, but Google Translate says that `neatitike` -> should be -> `neatitinka`

File: translations/messages+intl-icu.lt.xlf [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+                <source>help.more_information</source>
+                <target><![CDATA[Norėdami gauti daugiau informacijos, peržiūrėkite <a href="https://symfony.com/doc">Symfony dokumentciją</a>.]]></target>
+            </trans-unit>
+            <trans-unit id="help.post_summary">
+                <source>help.post_summary</source>
+                <target>Santrauka negali būti su Markdown ar HTML žymom, tiktai paprastas tekstas</target>
+            </trans-unit>
+            <trans-unit id="help.post_publication">
+                <source>help.post_publication</source>
+                <target>Nustatykite datą ateityje, kad suplanuotumėte blogo įrašo paskelbimą.</target>
```
Comment (javiereguiluz) [inline translations/messages+intl-icu.lt.xlf]: Probably a false positive, but Google Translate says that `suplanuotumėte` -> should be -> `planuotumėte`

File: translations/messages+intl-icu.lt.xlf [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+                <source>help.role_admin</source>
+                <target>administratorius</target>
+            </trans-unit>
+            <trans-unit id="help.reload_fixtures">
+                <source>help.reload_fixtures</source>
+                <target>Jei vartotojai neveikia, perleiskite šia komandą:</target>
+            </trans-unit>
+            <trans-unit id="help.add_user">
+                <source>help.add_user</source>
+                <target>Jeigu norite sukurti naują vartotoją paleiskite šia komandą:</target>
```
Comment (javiereguiluz) [inline translations/messages+intl-icu.lt.xlf]: Probably a false positive, but Google Translate says that `Jeigu` -> should be -> `Jei`

**Comment by uikolas** [comment]
Thanks for review! Maybe there are more simple mistakes, but in the future will be fixed by me, and other users.

**Comment by javiereguiluz** [comment]
Thanks @uikolas for working on this feature, this is much appreciated.

**Pull request closed without merging**